exercises/say/say_test.exs

```diff
@@ -0,0 +1,89 @@
+if !System.get_env("EXERCISM_TEST_EXAMPLES") do
+  Code.load_file("say.exs", __DIR__)
+end
+
+ExUnit.start
+ExUnit.configure exclude: :pending, trace: true
+
+defmodule SayTest do
+  use ExUnit.Case
+
+  test "says zero in english" do
```

Comment:
Typically we actually include the `@tag` line before the first test and comment it out. It's a nice indicator to the user as to how they're supposed to move through the test. Think you could add this?